My Simple App/Business Logic/AnyReactableResource.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AnyReactableResource.swift
+//  My Simple App
+//
+//  Created by Erick Sanchez on 8/12/20.
+//  Copyright © 2020 Erick Sanchez. All rights reserved.
+//
+
+import Foundation
+
+class AnyReactableResource<Resource>: ReactableResource {
+  var resource: Resource {
+    return base.resource
+  }
+
+  var didChangeEvent: Event<Void> {
+    return base.didChangeEvent
+  }
+
+  private let base: AbstractReactableResource<Resource>
+
+  init<T: ReactableResource>(base: T) where T.Resource == Resource {
```

Comment:
woodward4422 2 days ago @woodward4422
This is so abstract lol

LinnierGames 38 minutes ago Author Owner
This was very annoying to figure out haha